### DISCUSSION TITLE
Need to get all Topics, a Device publish

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#9184] [General]
Hello @Koenkk ,

in the legacy api, there is a possibility to GET all possible topics from one device. If I pull of legacy api, that will not run.
Is there a possibility to do that without having legacy api activated?

Background: 
For Symcon, we used this way, to implement a Devices via a configurator into Symcon:
![image](https://user-images.githubusercontent.com/20765982/137612832-18b5d809-a77c-4c0c-9fc8-56f9fa2f104d.png)

We used for that zigbee2mqtt/bridge/config/devices/get. In the new api, i know, that`s deprecated, but is there  an alternative way to get the topics from one device in the new api?

I readed all in your documentation and also I readed all in the called issues to that topic, but I didn't find any helpfull answer.

Many thanks for your support,

Greetings Burkhard

**Comment by Koenkk** [comment]
`bridge/devices` is a retained message (https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgedevices). So upon connecting to the MQTT broker the latest version will always be available in your application.
